test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -113,6 +113,15 @@ class ClusterManagerImplTest : public testing::Test {
         factory_.local_info_, log_manager_, factory_.dispatcher_));
```

Comment:
Do you also want to modify `ads_integration_test` to force warming behavior?

Reply:
The ADS integration test already does warming AFAICT because it sends a new CDS response followed by EDS. Do you mean to just verify warming by waiting on warming stats? Yes I can add this.